Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ft-lockup"
-version = "1.0.0"
+version = "2.0.0"
```

Comment:
I would not tick it to 2.0 here since 1.0 is hopefully not in use yet.

Reply:
@evgenykuzyakov do you agree that we should NOT bump the version?
PS: bumped initially because the state schema has been changed in an incompatible way.